describe('failing test', () => {
  it('should throw', () => {
    throw new Error('oops');
  });
});
